rs-matter/src/mdns/astro.rs

```diff
@@ -91,12 +90,16 @@ impl<'a> MdnsService<'a> {
     }
 
     /// Only for API-compatibility with builtin::MdnsRunner
-    pub async fn run_udp(&mut self, buffers: &mut MdnsUdpBuffers) -> Result<(), Error> {
+    pub async fn run_udp(&mut self, _buffers: &mut MdnsUdpBuffers) -> Result<(), Error> {
         core::future::pending::<Result<(), Error>>().await
     }
 
     /// Only for API-compatibility with builtin::MdnsRunner
-    pub async fn run(&self, _tx_pipe: &Pipe<'_>, _rx_pipe: &Pipe<'_>) -> Result<(), Error> {
+    pub async fn run<D>(
```

Comment:
Needs to be named `run_piped` now. I've swapped the name, as I expect folks to be using much more often the above method, than this one (which is for advanced use cases with a custom UDP stack).

Reply:
To clarify, the other method - say - this one - should have this signature:
```
    pub async fn run_piped(&self, _tx_pipe: &Pipe<'_>, _rx_pipe: &Pipe<'_>) -> Result<(), Error> {
```

Reply:
Look in `builtin::MdnsService` as how the method signatures (and names) should look like as this is what we are trying to match here.